android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
If the app is in the background and you're sending data and notification payload, the data is not received from onMessageReceived (as expected and documented by firebase). Notification is stored in system tray and Data in extras of the intent.